File: src/codes/Users_Manual/link/visualizer_phase.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Gain;
~using AUTD3Sharp.Modulation;
~using AUTD3Sharp.Utils;
var autd = Controller.Builder([new AUTD3(Vector3.Zero)]).Open(Visualizer.Builder());

var center = autd.Geometry.Center + new Vector3(0, 0, 150);
var g = new Focus(center);
autd.Send(g);

autd.Link.PlotPhase(new PlotConfig
{
    Fname = "phase.png"
}, autd.Geometry, Segment.S0, 0);

File: src/codes/Users_Manual/link/visualizer_phase.py
~import numpy as np
~from pyautd3 import AUTD3, Controller, Segment
~from pyautd3.gain import Focus
from pyautd3.link.visualizer import PlotConfig, Visualizer

autd: Controller[Visualizer] = (
    Controller.builder().add_device(AUTD3([0.0, 0.0, 0.0])).open(Visualizer.builder())
)

center = autd.geometry.center + np.array([0, 0, 150])
g = Focus(center)
autd.send(g)

autd.link.plot_phase(PlotConfig(fname="phase.png"), autd.geometry, Segment.S0, 0)